docs/faq.md

```diff
@@ -1,123 +1,150 @@
-FAQ
-==========
-
-版本相关
-""""""""""""""""
-
-:问:
-  FISCO BCOS v2.0与之前版本有哪些变化？
-:答:
-  请 :doc:`参考这里 <whats_new>`。
-
-:问:
-  FISCO BCOS v2.0都有哪些API？开发者如何与FISCO BCOS平台交互？
-:答:
-  1. FISCO BCOS v2.0提供jsonrpc接口，具体请 :doc:`参考这里 <api>`。
-  2. FISCO BCOS v2.0提供Java SDK帮助开发者快速实现应用，具体请 :doc:`参考这里 <sdk/index>`。
-  3. FISCO BCOS v2.0提供控制台帮助用户快速了解使用FISCO BCOS，具体请 :doc:`参考这里 <manual/console>`。
-
-:问:
-  FISCO BCOS v2.0如何搭建？
-:答:
-  - 建链脚本：使用简单，适合第一次接触FISCO BCOS的同学，具体请 :doc:`参考这里 <manual/build_chain>`。
-  - FISCO-Generator：更灵活，适合熟悉FISCO BCOS的用户和企业用户，具体请 :doc:`参考这里 <enterprise/index>`。
-
-:问:
-  FISCO BCOS v2.0的智能合约与之前版本合约有什么不同，兼容性如何？
-:答:
-  FISCO BCOS v2.0支持最新的Solidity合约，同时增加了precompare合约，具体请 :doc:`参考这里 <developer/index>`。
-
-:问:
-  国密和普通版本的区别有哪些？
-:答:
-  编译版本，证书，落盘加密，solidity编译java，web3sdk使用，具体请 :doc:`参考这里 <manual/guomi>`。
-
-:问:
-  是否支持从1.3或1.5升级到2.0版本？  
-:答:
+# FAQ
+
+## 版本相关
+
+问:
+  FISCO BCOS v2.0与之前版本有哪些变化？<br>
+答:
+  请 [参考这里](./whats_new)。
+
+问:
+  FISCO BCOS v2.0都有哪些API？开发者如何与FISCO BCOS平台交互？<br>
+答:
+  FISCO BCOS的API为
+  1. FISCO BCOS v2.0提供jsonrpc接口，具体请 [参考这里](./api)。
+  2. FISCO BCOS v2.0提供Java SDK帮助开发者快速实现应用，具体请 [参考这里](./sdk/index.html)。
+  3. FISCO BCOS v2.0提供控制台帮助用户快速了解使用FISCO BCOS，具体请 [参考这里](./manual/console.md)。
+
+问:
+  FISCO BCOS v2.0如何搭建？<br>
+答:
+  FISCO BCOS支持多种搭建方式，常用方式有：
+  - 建链脚本：使用简单，适合第一次接触FISCO BCOS的同学，具体请 [参考这里](./manual/build_chain.md)。
+  - FISCO-Generator：更灵活，适合熟悉FISCO BCOS的用户和企业用户，具体请 [参考这里](./enterprise/index.html)。
+
+问:
+  FISCO BCOS v2.0的智能合约与之前版本合约有什么不同，兼容性如何？<br>
+答:
+  FISCO BCOS v2.0支持最新的Solidity合约，同时增加了precompare合约，具体请 [参考这里](developer/index.html)。
+
+问:
+  国密和普通版本的区别有哪些？<br>
+答:
+  编译版本，证书，落盘加密，solidity编译java，web3sdk使用，具体请 [参考这里](./manual/guomi.md)。
+
+问:
+  是否支持从1.3或1.5升级到2.0版本?<br>
+答:
   不支持。
 
-控制台
-""""""""""""""""
-:问:
-  控制台指令区分大小写吗？
-:答:
+## 控制台
+
+问:
+  控制台指令区分大小写吗？<br>
+答:
   只支持小写。
 
-:问:
-  加入记账列表或观察者列表报错，nodeID is not in network，为什么？
-:答:
+问:
+  加入记账列表或观察者列表报错，nodeID is not in network，为什么？<br>
+答:
   节点加入记账列表和观察者列表的节点必须是连接peer的nodeID列表里面的成员。
 
-:问:
-  删除节点操作报错，nodeID is not in group peers，为什么？
-:答:
+问:
+  删除节点操作报错，nodeID is not in group peers，为什么？<br>
+答:
   节点删除操作中的节点必须是getGroupPeers里面展示的group的peers。
 
-:问:
-  游离节点（非群组节点）是否可以同步group数据？
-:答:
+问:
+  游离节点（非群组节点）是否可以同步group数据？<br>
+答:
   游离节点不参与group内的共识、同步和出块，游离节点可以通过am/ao命令可以将退出的节点添加为共识/观察节点。
 
-:问:
-  统一节点属于不同的group，是否可以支持查询多group的信息。
-:答:
-   可以，在进入控制台时，输入要查看的groupID:  ./web3sdk -c [groupID]
+问:
+  某节点属于不同的group，是否可以支持查询多group的信息。<br>
+答:
+   可以，在进入控制台时，输入要查看的groupID:  ./start [groupID]
 
-FISCO BCOS使用
-""""""""""""""""
+## FISCO BCOS使用
 
-:问:
-  系统配置、群组配置、节点配置分别指什么？
-:答:
+问:
+  系统配置、群组配置、节点配置分别指什么？<br>
+答:
   系统配置是指节点配置中一些影响账本功能，并需账本节点共识的配置项。群组配置指节点所属的群组的相关配置，节点的每个群组都有独立的配置。节点配置指所有可配置项。
```

Comment:
Maybe should give examples of group configuration(group.${group_id}.genesis and group.${group_id}.ini), node configuration(config.ini)